### DISCUSSION TITLE
Improving Github Profile ?????  : '|

### DISCUSSION THREAD

**Discussion by diva-21** [community/community#35533] [Other Feature Feedback, Questions, & Ideas] [answered]
Can anyone tell me on how to improve my github profile , i saw many profiles of developers, they are very good and i personally liked their work and contributions. Could you please tell me few suggestions and projects pleasee

**Comment by caioagiani** [comment]
You can improve your profile by perfecting and publishing all your personal/professional projects/README.

You can also contribute to the Open Source community by submitting PRs or opening issues that help improve something.

Another way is to get Badges to make your profile more attractive, here I teach you how to get an easy achievement: **How you can get the Public Sponsor Badge** #19916

**Reply by diva-21** [reply]
thank you @caioagiani , i will note of them and build stuff

**Comment by mwong3la** [comment]
Describe yourself.What you do.Which languages youre perfect in and include your projects.Create README.md for repositiries.Describe the repository inside the README.In my perspective that stands out.

**Reply by diva-21** [reply]
thank you @4ouR04 , kind of you

**Comment by tinniaru3005** [comment, marked as answer]
1. Have a good and personalized README.md 
- [GitHub docs for personalized README](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)
- [Eddie Jaoude course](https://www.eddiejaoude.io/course-github-profile-landing)
2. Commits
- Commit small changes
- Good commit messages describing what the commit is for
- Commit related changes together
- Commit often
- Don't Commit Half-Done Work
- Use Branches
3. Code
- Good README.md is a must
- The code must not contain duplicate and/or copied lines
- Write clean, commented, and well-organized code
- create functions that make your code more structured
5. Collaboration
- Allow others to contribute to your project
- Raise an issue describing the changes you want
- Allow others to send a PR solving the issue
6. Community
- Contribute to open-source communities
- Get involved in discussions
- Join the chat groups of the communities
- Try solving the issues on the repositories with the label `good-first-issue`
- Participate in open-source programs
7. PRs
- Have patience: The review often takes longer
- Have your work broken down into small commits. Make the work of the reviewer easy
- When creating a pull request you should care about the title and the description as a way to provide an overview of why the work is taking place with any relevant links.
- Do not make assumptions, ask for clarification.
8. Reviewing PRs
- It is a great way to contribute to any project
- Use emojis to clarify tone. Compare “Looks good :)” or “Looks good! 👍 ” with “It’s fine.”
- Offer ways to simplify or improve code
- Do not change the code while reviewing
9. Join organizations
10. Badges
- [Achievement Badges](https://drknzz.github.io/GitHub-Achievements/)

**Reply by diva-21** [reply]
woww, detailed explanation . these kind will also help other beginners, thank you @tinniaru3005 .
Loved your profile :)

**Reply by tinniaru3005** [reply]
You are welcome :)

**Comment by amber-moe** [comment]
You create a repo that shares the same name as your GitHub username. Then add a `README.md` file. Put your introductions in it and voila~~ Just created one https://github.com/Amber1990Zhang.

**Comment by ArnabDey1543** [comment]
> how to improve my github profile

You can change the position they appear in by dragging and dropping them. When you click into each repository, you can add a short description that will be visible on your profile, so you want to add something that tells them a little about the project and piques their interest.
